controllers/metal3cluster_controller.go

```diff
@@ -100,7 +101,7 @@ func (r *Metal3ClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 	clusterLog = clusterLog.WithValues("cluster", cluster.Name)
 
 	// Return early if BMCluster or Cluster is paused.
-	if util.IsPaused(cluster, metal3Cluster) {
+	if annotations.IsPaused(cluster, metal3Cluster) {
```

Comment:
This was already [taken](https://github.com/metal3-io/cluster-api-provider-metal3/pull/167/files#diff-250f2503d81624c7304619762d56ee502ff1462e5ad0c965af1eb9bfed5f340cR178) care in one of the `/hold` patches we had AFAIR:)

/lgtm

Reply:
Ops, didn't know that. I can close this